packs/spell-effects/spell-effect-sages-curse-failure.json

```
@@ -0,0 +1,60 @@
+{
+    "_id": "wzmPGA1HTgNvvKmp",
+    "img": "systems/pf2e/icons/spells/savants-curse.webp",
+    "name": "Spell Effect: Sage's Curse (Failure)",
+    "system": {
+        "description": {
+            "value": "<p>The target must roll twice and use the worse result whenever attempting an Arcana, Lore, Occultism, or Society check.</p>"
+        },
+        "duration": {
+            "expiry": null,
+            "sustained": false,
+            "unit": "unlimited",
+            "value": -1
+        },
+        "level": {
+            "value": 1
+        },
+        "publication": {
+            "license": "OGL",
+            "remaster": false,
+            "title": ""
```

```suggestion
            "title": "Pathfinder Abomination Vaults Adventure Path"
```
Also needs updated on the spell itself.